ts/session/apis/open_group_api/sogsv3/sogsApiV3.ts

```diff
@@ -405,7 +406,13 @@ async function handleInboxOutboxMessages(
         id: v4(),
         type: SignalService.Envelope.Type.SESSION_MESSAGE, // this is not right, but we forward an already decrypted envelope so we don't care
       };
-
+      const contentDecrypted = SignalService.Content.decode(content);
+      if (!shouldProcessContentMessage(builtEnvelope, contentDecrypted, true)) {
+        window.log.warn(
+          `received inbox/outbox message that did not pass the shouldProcessContentMessage test envelopeTs: ${builtEnvelope.timestamp}`
```

Comment:
We might need more information than just the timestamp to find this bad message right?